examples/data-sources/opslevel_group/data-source.tf

```diff
@@ -1,7 +0,0 @@
-data "opslevel_group" "foo" {
```

Comment:
Should this be kept for now?
I wonder if devs migrating their groups to teams might use this data source during that process.

Reply:
My thoughts are that the users who are using groups should have been informed as of the last release that groups are going away. So to discourage new users from using groups I think we should get rid of them from examples ASAP.